### PR TITLE
Add a blog about treehouses remote

### PR DESCRIPTION
**Pull request by empeje**
To test:

```
docker-compose up -d
```

### CODE REVIEW

**Review by lmmrssa — APPROVED** [review]
Looks nice as always

File: _posts/2018-05-05-moodlemootph18.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: treehouses.io in MoodleMoot Philippines 2018
 layout: post
-author: mappuji
+author: empeje
```
Comment (dogi) [inline _posts/2018-05-05-moodlemootph18.md]: @empeje ? please make an different issue/PR for both name changes they can be in one but not here ;)

File: _posts/2019-01-27-try-treehouses-remote.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Have you ever imagine that you can bring your battery-powered RaspberryPi in your bag and you can control it via an Android phone? Then you can do a fancy hack. My friend ever did something similar with a laptop plus an SDR (Software Defined Radio) device to hijack his friend remote controlled car door. But image now you can do the prank with the your Android phone.
+
+We inspired by our systems administrator and operator in the field who more-likely familiar with phone rather than advance computer skills. Which expected because the one who operate our software and hardware is volunteer working in a remote area. We want to make our software easy to operate. So instead of telling them how to do an SSH and fancy terminal, we create an app that can connect to the RaspberryPi as simple as pairing our Bluetooth headset to our phone.
+
+## Getting Started
+
+Requirements:
+* Android Phone
+  - [ ] Install [treehouses/remote](https://github.com/treehouses/remote/releases) app
+* RaspberryPi 3B or higher (any version with bluetooth)
```
Comment (dogi) [inline _posts/2019-01-27-try-treehouses-remote.md]: tested on
- 3B
- zero W (and WH)
- 3B+
- 3A+

File: _posts/2019-01-27-try-treehouses-remote.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  - [ ] Install [treehouses/builder][GETTING_STARTED] image (in this example we use treehouses image version `75`)
```
Comment (dogi) [inline _posts/2019-01-27-try-treehouses-remote.md:21]: - dev.ole.org or faster on cloudflare ... get our latest link here

File: _posts/2019-01-27-try-treehouses-remote.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+* A laptop
+  - [ ] Install [Etcher](https://www.balena.io/etcher/)
+  - [ ] Install Tor Browser (we have a fun demonstration)
```
Comment (dogi) [inline _posts/2019-01-27-try-treehouses-remote.md]: :D

Comment (dogi) [reply _posts/2019-01-27-try-treehouses-remote.md]: we should add link to torbrowser download

**Review by dogi — CHANGES_REQUESTED** [review]
have to stop here ;)
or still finished
but very good start

File: _posts/2018-05-05-moodlemootph18.md (hunk 1 above)
Comment (empeje) [reply _posts/2018-05-05-moodlemootph18.md]: sure, done here https://github.com/treehouses/treehouses.github.io/pull/36

File: _posts/2019-01-27-try-treehouses-remote.md (hunk 4 above)
Comment (empeje) [reply _posts/2019-01-27-try-treehouses-remote.md]: updated

File: _posts/2019-01-27-try-treehouses-remote.md (hunk 3 above)
Comment (empeje) [reply _posts/2019-01-27-try-treehouses-remote.md:21]: cloudfront you mean? what is the link @dogi?

File: _posts/2019-01-27-try-treehouses-remote.md (hunk 2 above)
Comment (empeje) [reply _posts/2019-01-27-try-treehouses-remote.md]: updated, please check @dogi

**Comment by empeje** [comment]
Hi @dogi I close this PR as I think things has changed in the Treehouses wiki.

**Pull request closed without merging**